packages/teleport/src/Apps/AppList/AppList.tsx

```diff
@@ -52,7 +55,7 @@ function Item({ app, ...rest }) {
         tabIndex={-1}
         target="_blank"
         color="text.primary"
-        href={app.addr}
+        href={app.activationUrl}
```

Comment:
`activationUrl` sounds kind of misleading, that this app needs to be activated/verfified. Maybe we can call it `launchUrl`?